adam/language_specific/english/english_language_generator.py

```diff
@@ -96,8 +100,25 @@ class _Generation:
         ] = dict()
 
         def generate(self) -> ImmutableSet[LinearizedDependencyTree]:
-            for _object in self.situation.objects:
-                self._translate_object_to_noun(_object)
+            # For now, only apply quantifiers to object-only situations
+            if self.situation.actions == ImmutableSet.empty():
```

Comment:
In Python, empty collections evaluate to `False`, so prefer:

```
if not self.situation.actions:
 ...
else:
  ...
```

You could also consider reversing the branches to eliminate the negation (negation tends to confuse readers)

Reply:
Done